packages/router-plugin/src/router.state.ts

```diff
@@ -174,4 +179,30 @@ export class RouterState {
       this.navigationTriggeredByDispatch = false;
     }
   }
+
+  /**
+   * No sense to mess up the `setUpStateRollbackEvents` method as we have
+   * to perform this check only once and unsubscribe after the first event
+   * is triggered
+   */
+  private checkInitialNavigationOnce(): void {
+    // Need to perform this check as `location` is not available
+    // on the server side
+    if (isPlatformServer(this.platformId)) {
+      return;
+    }
+
+    const isNotKarma = location.pathname !== '/context.html';
```

Comment:
Team, this should be removed, but needs discussion first. Karma manually enters `/context.html` URL, thus unit tests fails, because the router invokes `navigateByUrl('/context.html')`

Reply:
I will use `isAngularInTestMode` after [this one is merged](https://github.com/ngxs/store/pull/921)

Reply:
After exposing `isAngularInTestMode` - I will also try add some tests for that fix